src/Trinity.TSL/Trinity.TSL.CodeTemplates/Trinity.TSL.CodeTemplates.csproj

```diff
@@ -9,4 +9,12 @@
   <ItemGroup>
     <ProjectReference Include="../../Trinity.Core/Trinity.Core.csproj" />
   </ItemGroup>
+
+  <!-- Package dependencies -->
+  <ItemGroup>
```

Comment:
adding deps to the code template project -- it seems dotnet 7.0 toolchain does not propagate the deps from Trinity.Core anymore